db/queries/assessment_records/_helpers.py

```diff
@@ -1,9 +1,12 @@
+import jsonpath_rw_ext
```

Comment:
We need to add this dependency to the requirements file.

I am getting this error:
```
ModuleNotFoundError: No module named 'jsonpath_rw_ext'
```